Scripts/WaitAnimation.cs

```diff
@@ -8,7 +8,7 @@ public class WaitAnimation : TimedAnimation
 		/// <summary>
 		/// WaitAnimation only valid when the duration > 0
```

Comment:
I know no one reads the documentation, but you might wanna change it there as well...